package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "lint": "./node_modules/.bin/standard",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "jest"
```

Comment:
Is this just to make the circleci pass? There are not tests yet right?

Reply:
no tests yet, correct